### ISSUE TITLE
UnregisteredEnv: No registered env with id: highway-fast-v0

### ISSUE THREAD

**Issue by freemedom** [eleurent/highway-env#435]
https://github.com/eleurent/highway-env/blob/b621f9a79b3c09d107862c5ca19bf85e8923bedf/scripts/sb3_highway_dqn.ipynb?short_path=5c9a831#L126

**Comment by freemedom** [comment]
----> 1 model = DQN('MlpPolicy', "highway-fast-v0",
      2                 policy_kwargs=dict(net_arch=[256, 256]),
      3                 learning_rate=5e-4,
      4                 buffer_size=15000,
      5                 learning_starts=200,

[/usr/local/lib/python3.9/dist-packages/stable_baselines3/dqn/dqn.py](https://localhost:8080/#) in __init__(self, policy, env, learning_rate, buffer_size, learning_starts, batch_size, tau, gamma, train_freq, gradient_steps, replay_buffer_class, replay_buffer_kwargs, optimize_memory_usage, target_update_interval, exploration_fraction, exploration_initial_eps, exploration_final_eps, max_grad_norm, tensorboard_log, policy_kwargs, verbose, seed, device, _init_setup_model)
     95         _init_setup_model: bool = True,
     96     ):
---> 97         super().__init__(
     98             policy,
     99             env,

[/usr/local/lib/python3.9/dist-packages/stable_baselines3/common/off_policy_algorithm.py](https://localhost:8080/#) in __init__(self, policy, env, learning_rate, buffer_size, learning_starts, batch_size, tau, gamma, train_freq, gradient_steps, action_noise, replay_buffer_class, replay_buffer_kwargs, optimize_memory_usage, policy_kwargs, tensorboard_log, verbose, device, support_multi_env, monitor_wrapper, seed, use_sde, sde_sample_freq, use_sde_at_warmup, sde_support, supported_action_spaces)
    103         supported_action_spaces: Optional[Tuple[spaces.Space, ...]] = None,
    104     ):
--> 105         super().__init__(
    106             policy=policy,
    107             env=env,

[/usr/local/lib/python3.9/dist-packages/stable_baselines3/common/base_class.py](https://localhost:8080/#) in __init__(self, policy, env, learning_rate, policy_kwargs, tensorboard_log, verbose, device, support_multi_env, monitor_wrapper, seed, use_sde, sde_sample_freq, supported_action_spaces)
    157         # Create and wrap the env if needed
    158         if env is not None:
--> 159             env = maybe_make_env(env, self.verbose)
    160             env = self._wrap_env(env, self.verbose, monitor_wrapper)
    161 

[/usr/local/lib/python3.9/dist-packages/stable_baselines3/common/base_class.py](https://localhost:8080/#) in maybe_make_env(env, verbose)
     54         if verbose >= 1:
     55             print(f"Creating environment from the given name '{env}'")
---> 56         env = gym.make(env)
     57     return env
     58 

[/usr/local/lib/python3.9/dist-packages/gym/envs/registration.py](https://localhost:8080/#) in make(id, **kwargs)
    233 
    234 def make(id, **kwargs):
--> 235     return registry.make(id, **kwargs)
    236 
    237 

[/usr/local/lib/python3.9/dist-packages/gym/envs/registration.py](https://localhost:8080/#) in make(self, path, **kwargs)
    126         else:
    127             logger.info("Making new env: %s", path)
--> 128         spec = self.spec(path)
    129         env = spec.make(**kwargs)
    130         return env

[/usr/local/lib/python3.9/dist-packages/gym/envs/registration.py](https://localhost:8080/#) in spec(self, path)
    201                 )
    202             else:
--> 203                 raise error.UnregisteredEnv("No registered env with id: {}".format(id))
    204 
    205     def register(self, id, **kwargs):

UnregisteredEnv: No registered env with id: highway-fast-v0

---
I guess it has to do with gym and the gymnasium version.

**Comment by eleurent** [comment]
Duplicate of https://github.com/eleurent/highway-env/issues/431, this should now be fixed (please upgrade to 1.8.1)
